cypress/integration/userCanSeeCollectioOfArticle.feature.js

```diff
@@ -0,0 +1,27 @@
+describe('user can see a collectino of articles', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'https://reqres.in/api/users', {
+      fixture: 'articles.json',
+    }).as('getArticles')
+    cy.visit('/')
+  })
+
+  it('is expected to display a collection of 3 articles', () => {
+    cy.get('[data-cy=articles-list]').children().should('have.length', 3)
+  })
+
+  it('is expected that the first title will be Deep Work', () => {
+    cy.get('[data-cy=articles-list]')
+      .first()
+      .should('have.contain', 'Deep Work')
+  })
+
+  it('is expected that the first title have a clickable URL', () => {
+    cy.get('[data-cy=articles-list]')
+      .last()
+      .should('have.contain', 'Tipping Point')
+  
```

Comment:
Format code, and this is not testing for clickable URL, it is (maybe) testing for the title of the last article. Consider removing or redrafting.

Reply:
Revised and formatted code. Will include clickable URL test in the next feature.